src/Feature/Checkout/client/Integration/OrderHistory/sagas.ts

```diff
@@ -30,15 +30,14 @@ export function* getOrderHistory(requestData: Action<OrderHistoryRequestPayload>
   try {
     yield put(actions.GetOrderHistoryRequest());
     const firstPageNumber = 0;
-    const { data, error }: Result<Commerce.OrderHistoryResultModel> = yield call(Order.getOrders, firstPageNumber, ItemsPerPage);
+    const { data, error }: Result<Commerce.Order[]> = yield call(Order.getOrders, firstPageNumber, ItemsPerPage);
 
     if (error) {
       return yield put(actions.GetOrderHistoryFailure(error.message || 'Error Occured'));
     }
 
-    const orders = data.orders as Commerce.OrderModel[];
-    const isLastPage = !orders || orders.length < ItemsPerPage;
-    yield put(actions.GetOrderHistorySuccess(orders, data.currentPageNumber, isLastPage));
+    const isLastPage = !data || data.length < ItemsPerPage;
```

Comment:
Will we see "LOAD MORE" button if count of orders on last page equal to items per page?